packages/tools/pie-icons-configs/__tests__/configs.test.js

```diff
@@ -110,126 +112,136 @@ describe('validateGetRegularIconSize', () => {
+        it('returns an object with default icon width and height properties', () => {
+            const received = getSvgProps('icon-test', null, null);
 
-                expect(spy).toHaveBeenCalled();
-                expect(spy).toHaveBeenCalledWith(expect.stringContaining('Invalid prop "iconSize" value'));
-                expect(spy).toHaveBeenCalledWith(expect.stringContaining(iconName));
-                spy.mockRestore();
-            });
+            expect(received.width).toEqual(defaultRegularIconSize);
+            expect(received.height).toEqual(defaultRegularIconSize);
+        });
+        it('does not output a console error', () => {
+            const spy = vi.spyOn(console, 'error').mockImplementation(vi.fn());
+
+            expect(spy).not.toHaveBeenCalled();
+            spy.mockRestore();
         });
     });
 
-    describe('when provided with a class for large size icon', () => {
-        describe('when provided with valid parameters', () => {
-            it('returns an object that contains a class and width and height properties', () => {
-                const received = getSvgProps('pie-icon pie-icon--app-order-large', null, largeIconSizeDefault);
+    describe('when svgClasses parameter contains -large in class name', () => {
+        describe('when provided with valid iconSizeValue parameter', () => {
+            it('returns an object that contains a width and height attr with the correct values', () => {
+                const received = getSvgProps('icon-test-large', null, largeIconSizeDefault);
 
-                expect(received).toHaveProperty('class');
                 expect(received).toHaveProperty('width');
                 expect(received).toHaveProperty('height');
-            });
-
-            describe('when provided with a staticClasses parameter', () => {
-                it('returns an object with the expected classes', () => {
-                    const received = getSvgProps('pie-icon pie-icon--app-order-large', 'FOO BAR', largeIconSizeDefault);
-
-                    expect(received.class).toEqual(expect.stringContaining('pie-icon'));
-                    expect(received.class).toEqual(expect.stringContaining('pie-icon--app-order'));
-                    expect(received.class).toEqual(expect.stringContaining('FOO BAR'));
-                });
+                expect(received.width).toEqual(largeIconSizeDefault);
+                expect(received.height).toEqual(largeIconSizeDefault);
             });
         });
 
-        describe('when provided with invalid parameters', () => {
-            describe('when provided with an invalid iconSize', () => {
-                const iconName = 'IconAppOrder';
-                const errorMock = vi.fn();
-                console.error = errorMock;
+        describe('when provided with invalid iconSizeValue parameter', () => {
+            const invalidSize = 49;
+            const expectedSize = 48;
```

Comment:
Interesting that if they provide an invalid value we try to fix it for them, should we not just return the default? It might make it a bit more obvious that they've provided an invalid value (besides the error).

Reply:
Yeah, it will align the behaviour between regular and large as well. I will add the change 👍

Reply:
Done